packages/design-tokens/src/default/scss/_colors.scss

```diff
@@ -31,123 +41,85 @@ $danger-600: #c95771 !default;
 $danger-800: #af4c62 !default;
 
 // Brand
-$brand-primary: #132930 !default;
-$brand-secondary: #77a6c0 !default;
+$brand-primary: $brand-550 !default;
+$brand-secondary: $brand-300 !default;
```

Comment:
What if `$brand-secondary` needs to be derived from a palette other than `$brand-X`? Like here, blue and orange:

![obrazek](https://user-images.githubusercontent.com/5614085/131818443-1614d99a-3188-4922-a4ab-3797bc297b6a.png)

Don't we need another graded palette for that?

Reply:
After discussion, we decided to use **color** scales on lvl 0 of config instead. Each product can set up their own and use them on 1st level of color config (brand, bg, text, etc.).